Src/LinearSolvers/MLMG/AMReX_MLABecLaplacian.H

```diff
@@ -1031,7 +1033,9 @@ MLABecLaplacianT<MF>::Fsmooth (int amrlev, int mglev, MF& sol, const MF& rhs, in
                 });
             }
         }
-        Gpu::streamSynchronize();
```

Comment:
I think at the end of Fsmooth we should add the following in case this is in nosync region.
```
if (Ax.local_size() > 0 && Gpu::inNoSyncRegion()) { Gpu::streamSynchronize(); }
```